src/test/ui/maybe-bounds.stderr

```diff
@@ -1,22 +1,22 @@
 error: `?Trait` is not permitted in supertraits
-  --> $DIR/maybe-bounds.rs:11:12
+  --> $DIR/maybe-bounds.rs:11:11
    |
 LL | trait Tr: ?Sized {} //~ ERROR `?Trait` is not permitted in supertraits
-   |            ^^^^^
```

Comment:
@topecongiro 
>Though I am not sure how I should test the new code

You can put any of `?Sized`s in this test into parentheses, that would be enough.